src/workers/parser/utils/ast.js

```diff
@@ -14,28 +14,28 @@ import { getSource } from "../sources";
 let ASTs = new Map();
 
 function _parse(code, opts) {
-  return babylon.parse(code, opts);
+  return babylon.parse(code, {
+    ...opts,
+    tokens: true
```

Comment:
This `_parse` function is also called from `parse-script-tags` so it needs `tokens: true` too. This was causing the mochitest failures. Even worse, that module assumes tokens will always be present, so instead of resulting in a `tokens` result of `undefined`, it returns `[undefined]` as an array with a single item.

Reply:
@JLast if `parse-script-tags` is actually something used anywhere else, you probably want to have it pass this option automatically, or update it to handle the case where the user didn't pass in `tokens: true`.